minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java

```diff
@@ -104,7 +104,8 @@ private LocalTachyonMaster(final String tachyonHome, TachyonConf tachyonConf)
     tachyonConf.set(Constants.HOST_RESOLUTION_TIMEOUT_MS, "250");
 
     tachyonConf.set(Constants.WEB_THREAD_COUNT, "1");
-    tachyonConf.set(Constants.WEB_RESOURCES, System.getProperty("user.dir") + "/src/main/webapp");
+    tachyonConf.set(Constants.WEB_RESOURCES, System.getProperty("user.dir")
+        + "/../servers/src/main/webapp");
```

Comment:
Actually, to have better portability, could we use `tachyon.util.io.PathUtils#concatPath` for this path calculation?